src/lib/gamepad.ts

```diff
@@ -47,24 +45,20 @@ function handleArrowDown() {
 function handleArrowLeft() {
   const prevButton = document.getElementById('previous') as HTMLButtonElement
   if (prevButton) {
-    prevButton.dispatchEvent(upPointerEvent)
+    prevButton.dispatchEvent(pointerUpEvent)
```

Comment:
now that i look at this... is the right thing to dispatch a const event, rather than minting a new one? I wonder if the latter isn't more "by the book".

Reply:
Just so that I understand… are you suggesting this…?

```diff
- prevButton.dispatchEvent(pointerUpEvent)
+ prevButton.dispatchEvent(new PointerEvent('pointerup', { bubbles: true }))
```

If so, what difference does it make? If not, what are you suggesting?

Reply:
That's what I'm suggesting yes. I'm wondering if there are internal event implementation reasons why they shouldn't be reused. Not certain, just being paranoid. Like what happens if the event is marked "no bubbling" at some later point?